README.md

```diff
@@ -133,6 +133,19 @@ submitToTreeherder(
 )
 ```
 
+## testProject
```

Comment:
The other entries are pipeline steps, whereas this is a ServiceBook class. It feels like there should be some separation in the documentation. Perhaps a heading for pipeline steps, and a heading for ServiceBook with the `testProject` method documentated? We're probably close to wanting to break the documentation out to a readthedocs or similar too, as it's getting quite lenghty.

Reply:
Took a stab at this, but currently the "Shared Libraries..." heading and "Pipeline Steps" and "ServiceBook Class" are all the same heading weight of "#" - let me know if that's a deal-breaker, and I'll rework the subheadings' weights to match!